home_work_4/converter.py

```diff
@@ -0,0 +1,19 @@
+
+EXCHANGE_RATES = {
+    "EUR": 50.1,
+    "USD": 45.1,
+    "UAN": 30.1
+}
+
+def check(currency):
+    if currency not in EXCHANGE_RATES: raise ValueError("Неизвестная валюта " + currency)
+
+def convert(curr_from, curr_to, amount):
+    check(curr_from); check(curr_to)
+    return amount * EXCHANGE_RATES[curr_from] / EXCHANGE_RATES[curr_to]
+
+cf = input('Currency1:')
+ct = input('Currency2:')
+amt = int(input('Quantity:'))
+
+print(convert(cf, ct, amt))
```

Comment:
I am not sure that it logically works correctly
![image](https://github.com/VladyslavKanarkin/home_work_Vlad_Kanarkin/assets/51053483/7e9f718c-22f6-4f06-9849-5bda84af74be)